managed_key/config_test.go

```diff
@@ -0,0 +1,47 @@
+//go:build hsm
```

Comment:
Not sure why this build tag exists within OSS? With this the test will never run within this build so maybe the test doesn't belong within OSS or we need to remove this build tag?